test/test_core.py

```diff
@@ -8035,6 +8035,24 @@ def test_setlocale(self):
   def test_vswprintf_utf8(self):
     self.do_run_in_out_file_test('vswprintf_utf8.c')
 
+  # Test that a main with arguments is automatically asyncified.
+  @no_wasm64('TODO: asyncify for wasm64')
+  @with_asyncify_and_stack_switching
+  def test_async_main(self):
+    # needs to flush stdio streams
+    self.set_setting('EXIT_RUNTIME')
```

Comment:
If you add a newline to the end of the printf then maybe you can remove this line?